mubench.reviewsite/js/tag-util.js

```diff
@@ -0,0 +1,47 @@
+function addTag(event){
+    if (event.key === "Enter") {
+        let misuseTagsDiv = document.getElementById('misuse-tags');
+        tagName = event.target.value;
+        if (misuseTagsDiv.querySelectorAll("span#" + tagName).length == 0) {
+            let msg = {
+                "tag_name": tagName,
+                "misuse_id": misuseId
+            };
+            postJson(addUrl, msg, function(r){
```

Comment:
The `addUrl` should be a parameter to this function and not a globally defined variable that is assumed to exist. For remove you already did this.